uk.co.compucorp.civicrm.hrleaveandabsences/scss/components/_leave-request-record-actions.scss

```diff
@@ -0,0 +1,30 @@
+leave-request-record-actions {
+  display: block;
+
+  .leave-request-record-actions__button {
+    display: block;
+    float: left;
+    margin-left: 0;
+
+    &:first-child {
+      margin-right: $crm-gap-medium;
+    }
+  }
+}
+
+@media screen and (max-width: $screen-xs-max) {
```

Comment:
```scss
leave-request-record-actions {
  // desktop

  @media screen and (max-width: $screen-xs-max) {
    // mobile
    .leave-request-record-actions__button { }
  }
}
```

Reply:
also i think it would be preferable if you used a mobile first approach

Reply:
> also i think it would be preferable if you used a mobile first approach

why?

Reply:
It's the approach that is considered, in the industry, the preferred one out of the two (the other being desktop-first). There are multiple resources out there explaining why, in more details and more competently that I could ever do, so https://www.google.com/search?q=mobile+first+approach&oq=mobile+first+approach

Reply:
I understand this is another approach, but I think it would not really be helpful in our case since we work on top of Bootstrap that follows "desktop first". This means in order to, for example, to make a button 100% width for the mobile we will need to:

Desktop first:
```css
/* We need only one style assignment since we already have default desktop values */
@media mobile {
  button {
    width: 100%;
  }
}
```

Mobile first:
/* We need two assignments since we will override default desktop values and then will need to override back */
```css
button {
  width: 100%;

  @media desktop {
    width: auto;
  }
}
```